db/migrations/0004_alter_movie_actors_alter_movie_genres_and_more.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.0.2 on 2024-11-05 16:53
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_alter_moviesession_cinema_hall_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='actors',
+            field=models.ManyToManyField(related_name='movies_actor', to='db.Actor'),
+        ),
+        migrations.AlterField(
+            model_name='movie',
+            name='genres',
+            field=models.ManyToManyField(related_name='movies_genre', to='db.Genre'),
+        ),
+        migrations.AlterField(
+            model_name='moviesession',
+            name='cinema_hall',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='movie_sessions_halls', to='db.cinemahall'),
```

Comment:
The `related_name='movie_sessions_halls'` is unnecessarily complex. Consider using `related_name='movie_sessions'` to maintain simplicity and clarity.